include/vrv/doc.h

```diff
@@ -98,6 +98,11 @@ class Doc : public Object {
      */
     bool GenerateHeader();
 
+    /**
+     * Generate the measure indices
+     */
+    void GenerateMeasureIndices();
```

Comment:
Rename `PrepareMeasureIndices` since to be called from `PrepareData` (see other comment)